<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsingPlanWidget</class>
 <widget class="QWidget" name="UsingPlanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="TitleLable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>启用计划</string>
   </property>
  </widget>
  <widget class="QLabel" name="PosLable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>780</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLable_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>查询条件：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="TitleLable_4">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Pos版本：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PosVerLe"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="TitleLable_5">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>最后心跳时间（大于）：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PosHeartBeatLe"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AllChooseBtn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>158</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="StartBtn">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>启用</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>SQL语句：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
